src/services/codefixes/fixAddMissingMember.ts

```diff
@@ -23,31 +29,41 @@ namespace ts.codefix {
             const seenNames = createMap<true>();
             return codeFixAll(context, errorCodes, (changes, diag) => {
                 const { program, preferences } = context;
-                const info = getInfo(diag.file, diag.start, program.getTypeChecker());
-                if (!info) return;
-                const { classDeclaration, classDeclarationSourceFile, inJs, makeStatic, token, call } = info;
-                if (!addToSeen(seenNames, token.text)) {
+                const checker = program.getTypeChecker();
+                const info = getInfo(diag.file, diag.start, checker);
+                if (!info || !addToSeen(seenNames, info.token.text)) {

```

Comment:
So... this is not really caused by your change, but was already there, but i think this is wrong.. the key to addToSeen should not be the name of the property, but rather the the pair [container symbol, property Name]. e.g. today this fails:

```ts
class C {
    method() { 
        this.a = 0;
    }
}


class D {
    method() {
        this.a = 0;
    } 
}   
```

it gets a bit more involved if one of the two classes inherit from each other.

Reply:
I think we can handle that in a different change since it was not introduced by your change.